kpi/management/commands/cypress_testserver.py

```diff
@@ -1,384 +0,0 @@
-            )
-
-        verbosity = options['verbosity']
-        interactive = options['interactive']
-
-        # Create a test database.
-        db_name = connection.creation.create_test_db(
-            verbosity=verbosity, autoclobber=not interactive, serialize=False
-        )
-
-        if fixture_labels:
-            # Optionally import any fixtures specified on the command line into
-            # the database.
-            call_command('loaddata', *fixture_labels, **{'verbosity': verbosity})
-        else:
-            # Load the `test_data` fixture by default
-            call_command('loaddata', ['test_data'], **{'verbosity': verbosity})
-
-        # Insert fixture data that are written here as Python code
-        self.insert_test_data()
-
-        # Run the development server. Turn off auto-reloading because it causes
-        # a strange error -- it causes this handle() method to be called
-        # multiple times.
-        shutdown_message = (
-            '\nServer stopped.\nNote that the test database, %r, has not been '
-            'deleted. You can explore it on your own.' % db_name
-        )
-        use_threading = connection.features.test_db_allows_multiple_connections
-        call_command(
-            'runserver',
-            addrport=options['addrport'],
-            shutdown_message=shutdown_message,
-            use_reloader=False,
-            use_ipv6=options['use_ipv6'],
-            use_threading=use_threading
-        )
-
-    @classmethod
-    def insert_test_data(cls):
-        """
-        Intended to provide for the following scenarios:
-
-        | USERNAME/PASSWORD  | DONE? | STATE NEEDED                                       |
-        | ------------------ | ----- | -------------------------------------------------- |
-        | project_creator    | Yes   | Empty                                              |
-        | question_creator   | Yes   | Has a project with no questions.                   |
-        | project_deleter    | Yes   | Has a deployed project.                            |
-        | question_deleter   | Yes   | Has a project that has a question.                 |
-        | template_creator   | Yes   | Has a project with questions.                      |
-        | template_cloner    | Yes   | Has a template                                     |
-        | project_replacer   | Yes   | Has a project and a template with different names. |
-        | rest_creator       | Yes   | Has a deployed project.                            |
-        | rest_editor        | No    | Has a deployed project with a REST service added.  |
-        | rest_deleter       | No    | Has a deployed project with a REST service added.  |
-        | submission_counter | No    | Has a deployed project with a REST service added.  |
-        | submission_retryer | No    | Has a deployed project with a REST service added.  |
-        """
```

Comment:
This looks important. Are we running cypress anywhere regularly? If we run it regularly, I'd rather postpone merging a bit until coordinated with QA what we are doing instead. :eyes: If we don't run cypress regularly and this effectively is a dead code, then no harm done removing it finally :ok_hand: